src/contracts/proxies/HaiSafeManager.sol

```diff
@@ -120,12 +125,16 @@ contract HaiSafeManager {
     _usrSafes[_usr].add(_safeId);
     _usrSafesPerCollat[_usr][_cType].add(_safeId);
 
+    IVault721(vault721).mint(_usr, _safeId);
+
     emit OpenSAFE(msg.sender, _usr, _safeId);
     return _safeId;
   }
 
   // Give the safe ownership to a dst address.
-  function transferSAFEOwnership(uint256 _safe, address _dst) external safeAllowed(_safe) {
+  function transferSAFEOwnership(uint256 _safe, address _dst) external {
```

Comment:
why isnt `safeAllowed(_safe)` allowed modifier needed here any more? 

let's add a test for this specifically

Reply:
`safeAllowed` allows the safe.owner or anyone authed on the safe to make modifications, including transfer. so we remove it here to prevent NFT delinking, and require that transfers can only be called via Vault721, which checks the safe.owner via NFT ownership